ui/TabConsole.qml.autosave

```diff
@@ -0,0 +1,7 @@
+import QtQuick 2.14
```

Comment:
You've committed a backup file here. The actual file `TabConsole.qml` Hasn't been updated. To resolve:

1. Reset this commit
2. Save the file fully
3. Commit the saved file
4. Add another commit that adds this type of file to the `.gitignore` so it doesn't appear in future. Line would be `*.autosave`